fjord/base/templates/404.html

```diff
@@ -1,16 +1,18 @@
 {% extends "base.html" %}
 
 {% block page_title %}{{ _('Not Found') }}{% endblock %}
-
+{% block site_css %}
+      {{ css('base') }}
+      <link rel="stylesheet" href="{{ settings.STATIC_URL }}css/style.css"/>
+    {% endblock %}
 {% block content %}
-  <div class="col"></div>
+ <div class="col"></div>
   <div class="col wide">
     <div class="block">
-      <h1>{{ _('Not Found') }}</h1>
-      <p>
-        {{ _('The document could not be found.') }}
-      </p>
-      <p>
+      <img id="image" src="{{ settings.STATIC_URL }}img/fjord404.jpg"/>
+      <p id="text">{{ _( 'The document could <br>not be found.' )}}</p>
```

Comment:
Another nit: "image" and "text" aren't very good identifiers. What are they? Why do they have identifiers instead of classes?